README.md

```diff
@@ -2,7 +2,7 @@
 List of top ten psychologists:
 
 1. B. F. Skinner
-1. Zigmund Fraud
+1. Ziggy Freud
```

Comment:
Are you sure this is spelled with double "g"?

Reply:
I'm not absolutely sure, but it's very likely to be spelled like this